### PR TITLE
initial tests created, passing, more CSS handled

### PR DESCRIPTION
**Pull request by ikheraj17**

### CODE REVIEW

File: client/__tests__/index.spec.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+
+describe('My Test Suite', () => {
+  it('My Test Case', () => {
+    expect(true).toEqual(true);
```
Comment (Joseph765) [inline client/__tests__/index.spec.js:5]: omg, sooo true!

File: client/index.jsx [hunk 2]
```diff
@@ -89,21 +114,97 @@ class App extends React.Component {
...
+          sIndex: start,
+          eIndex: end,
+          displayed: next,
+        })
+      }
+    }
+  }
+
+  onPreviousReviews() {
+    if(this.state.onProd === true) {
```
Comment (Joseph765) [inline client/index.jsx:154]: I'm curious about what you mean by 'onProd'?

**Review by Joseph765 — APPROVED** [review]
lgtm!

**Comment by Malyarc** [comment]
TGIW! LGTM!

File: client/index.jsx (hunk 2 above)
Comment (ikheraj17) [reply client/index.jsx:154]: I have a toggle in state called  "onProd" that is true if the product list is displayed, and becomes false when the shop reviews are displayed. This way, I can get a hold of the right arrays to slice and set to display when toggling between shop reviews and product reviews, and pressing the buttons to move forward or backwards through pages

File: client/__tests__/index.spec.js (hunk 1 above)
Comment (Malyarc) [reply client/__tests__/index.spec.js:5]: LMAO, LGTM though! Legit true equals to true. Can confirm 👍

**Pull request merged**